portfolio/src/main/webapp/script.js

```diff
@@ -13,10 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds "Hello Naa'Oli!" to the page.
+ * Adds greetings to the page.
  */
 function getHelloName() {
```

Comment:
i think you can remove the old version entirely. let's just keep one version of this function around.